--- origsrc/gforth-0.7.0/configure.in	2008-11-02 11:14:57.000000000 -0600
+++ src/gforth-0.7.0/configure.in	2009-01-08 22:29:21.782750000 -0600
@@ -333,14 +333,14 @@ echo "Check for arch/$machine/$platform/
 if test x$EC_MODE = xtrue
 then
 	echo "Check for arch/$machine/$platform/gforth.ld"
-	if test -f arch/$machine/$platform/gforth.ld
+	if test -f $srcdir/arch/$machine/$platform/gforth.ld
 	then
 		LDFLAGS="-T ../arch/$machine/$platform/gforth.ld -Map \$@.map -cref --gc-sections $LDFLAGS"
 		if test x$platform = xnxt; then
 			LIBS="$LIB_PATH/lib/gcc/arm-elf/$($CC --version | grep GCC | cut -d' ' -f3)/interwork/libgcc.a $LIB_PATH/arm-elf/lib/interwork/libc.a $LIBS"
 		fi
 	fi
-	if test -f arch/$machine/$platform/make.inc
+	if test -f $srcdir/arch/$machine/$platform/make.inc
 	then
 		MAKEINC="include ../arch/$machine/$platform/make.inc"
 	fi
